src/server/api/controllers/orders.controller.js

```diff
@@ -68,9 +68,9 @@ const getOrderByUserId = async (userid) => {
   }
   try {
     const orders = await knex('orders').where({ user_id: userid });
-    if (orders.length === 0) {
-      throw new Error(` order with the specified userid was not found`, 404);
-    }
+    // if (orders.length === 0) {
```

Comment:
Why did you comment this out?

Reply:
I want empty array .. so i wont get error
but when it is empty .. I am getting error from api and page stopped to load.. and also its not even in orderItems.error so i can check if it is an error ...Without this I am getting proptypes requirement for Purchases component

Reply:
fixed